src/PhpSpreadsheet/Reader/Xlsx.php

```diff
@@ -972,6 +972,10 @@ public function load($pFilename)
                                                 break;
                                         }
 
+                                        if (!$this->readEmptyCells && ($value === null || $value === '')) {
+                                            continue;
```

Comment:
`continue`ing here will make you skip the `$rowIndex += 1;` which is certainly not what we want. Update tests to cover that non-empty cells coordinates (and values) are correct even when skipping empty cells.